### ISSUE TITLE
Load plugin completions without aliases

### ISSUE THREAD

**Issue by remcohaszing** [ohmyzsh/ohmyzsh#7328]
Oh-my-zsh is great! It makes configuring zsh very easy.

However, it does a bit too much for my taste. Many plugins tend to add autocompletion for a command and a bunch of aliases. Personally I love to have that autocompletion, but I don’t like most of the aliases.

Is there a nice way to load only plugin completions, but not the aliases?

**Comment by dimensi0n** [comment]
Many plugins have only aliases.

**Comment by mcornella** [comment]
You could just add the path to the plugin folder in `$fpath` before OMZ is sourced, and take the plugin out of the plugins array. For example, if you want swiftpm completion, add
```
fpath+=($ZSH/plugins/swiftpm)
plugins=(....) # make sure swiftpm is not in here
source $ZSH/oh-my-zsh.sh
```
It's a temporary hack, but this requires much thought to implement in a lean fashion.

**Comment by mcornella** [comment]
There's a proposal for a way to turn off aliases in https://github.com/ohmyzsh/ohmyzsh/issues/10644. Please have a look and add your feedback. Thanks!

**Comment by mcornella** [comment]
This was implemented in #11550, see https://github.com/ohmyzsh/ohmyzsh#skip-aliases for how to use it.
